x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrailTests.java

```diff
@@ -249,11 +250,11 @@ public void testAnonymousAccessDeniedRest() throws Exception {
                 .put(LoggingAuditTrail.ORIGIN_TYPE_FIELD_NAME, LoggingAuditTrail.REST_ORIGIN_FIELD_VALUE)
                 .put(LoggingAuditTrail.ORIGIN_ADDRESS_FIELD_NAME, NetworkAddress.format(address))
                 .put(LoggingAuditTrail.REQUEST_METHOD_FIELD_NAME, request.method().toString())
-                .put(LoggingAuditTrail.REQUEST_BODY_FIELD_NAME,
-                        includeRequestBody && Strings.hasLength(expectedMessage) ? expectedMessage : null)
```

Comment:
Throughout these tests, there were several places where null values were being put into immutable maps, which the JDK rejects.